cmd/errors.go

```diff
@@ -10,7 +10,6 @@ var (
 	errInvalidBothCustomImageandVersion            = errors.New("you provided both a Custom image and a version. You have to provide only one of these to upgrade")                             //nolint
 
 	errConfigProjectName = errors.New("project name is invalid")
-	errProjectNameSpaces = errors.New("this project name is invalid, a project name cannot contain spaces. Try using '-' instead")
```

Comment:
Obsolete error message. Cobra will show help text if more than one argument is specified.